201-asev2-ilb-with-web-app/azuredeploy.json

```diff
@@ -0,0 +1,157 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "aseName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the App Service Environment"
+      }
+    },
+    "vnetResourceName": {
```

Comment:
need to split these up so that you have

vnetResourceGroupName
vnetName
subnetName

currently we're mixing matching names and resourceId()s - the simple approach is to prompt for the names and construct the ids

Reply:
done